lib/maths/CStatisticalTests.cc

```diff
@@ -74,6 +74,9 @@ double CStatisticalTests::leftTailFTest(double x, double d1, double d2) {
     if (std::isinf(x)) {
         return 1.0;
     }
+    if (std::isnan(x)) {
+        return 0.0;
```

Comment:
I would expect this to return the same value as for `isinf(x)` -> `1.0`?

Reply:
Yes, it makes sense to have them on the same footing